_layouts/home.html

```diff
@@ -20,6 +20,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="cleartype" content="on">
 
+    <link href="https://fonts.googleapis.com/css?family=Titillium+Web:600,700|Yantramanav" rel="stylesheet">
```

Comment:
This only affects the home page, I'll commit a small refactor to share the `<head>` with all pages, app-wide.